wled00/wled_server.cpp

```diff
@@ -112,13 +112,11 @@ void initServer()
     serveSettings(request);
   });
   
-  server.on("/settings.js", HTTP_GET, [](AsyncWebServerRequest *request){
-    serveSettingsJS(request);
-  });
+  // "settings.js" request also handled by serveSettings()
```

Comment:
This will not work in every situation. Tested.

Reply:
Thanks for letting me know, will check it out :)